src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```diff
@@ -2383,4 +2385,9 @@ public void visitCtReceiverParameter(CtReceiverParameter receiverParameter) {
 			printer.writeSeparator("this");
 		}
 	}
+
+	@Override
+	public void visitCtUnnamedPattern(CtUnnamedPattern unnamedPattern) {
+		printer.writeKeyword(CtLocalVariableImpl.UNNAMED_VARIABLE_NAME);
```

Comment:
Why does this constant live in CtLocalVariableImpl and not an interface? Multiple places can be unnamed (e.g. also catch variables which somehow are not local variables)...

Reply:
I wasn't sure if we want to expose it directly. It's rather our internal representation, but maybe it shouldn't be used here then...

Reply:
There is an `isUnnamed` getter, yea. The question is probably how people can create their own unnamed variables.

Reply:
The constant actually lives in the interface, I just forgot that already when I used it here...